server.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
```

Comment:
> ah ok, didn't know that

Now you know something new :-)


> It would make sense if this is a server, but since it isn't I vote for revert on this one

I do it this way when I have a single binary (the usual) so I can be able to exec them too with `npm start` as another entrypoint, but if you find it confusing I can be able to revent it too.